### ISSUE TITLE
Uncaught RangeError: Invalid array length

### ISSUE THREAD

**Issue by slang800** [NikhilKalige/docblockr#100]
1. enable "Per Section Indent"
2. paste this into the editor (in coffeescript):

  ```coffee
  ###*
  test = ->
  ```
3. place cursor after the asterisk, and hit enter/return
4. observe that the following error is thrown:

**Atom Version**: 0.198.0
**System**: linux 3.19.0-16-generic
**Thrown From**: [docblockr](https://github.com/nikhilkalige/docblockr) package, v0.7.1

### Stack Trace

Uncaught RangeError: Invalid array length

```
At /usrdata/.atom/packages/docblockr/lib/docblockr-worker.js:726

RangeError: Invalid array length
  at DocBlockrAtom.module.exports.DocBlockrAtom.repeat (/usrdata/.atom/packages/docblockr/lib/docblockr-worker.js:726:27)
  at DocBlockrAtom.module.exports.DocBlockrAtom.align_tags (/usrdata/.atom/packages/docblockr/lib/docblockr-worker.js:943:44)
  at DocBlockrAtom.module.exports.DocBlockrAtom.generate_snippet (/usrdata/.atom/packages/docblockr/lib/docblockr-worker.js:834:20)
  at DocBlockrAtom.module.exports.DocBlockrAtom.parse_command (/usrdata/.atom/packages/docblockr/lib/docblockr-worker.js:280:26)
  at atom-workspace.<anonymous> (/usrdata/.atom/packages/docblockr/lib/docblockr-worker.js:75:16)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app.asar/src/command-registry.js:238:29)
  at /usr/share/atom/resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:519:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:354:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/usr/share/atom/resources/app.asar/src/window-event-handler.js:178:20)

```

### Commands

```
     -1:03.4.0 docblockr:parse-enter (atom-text-editor.editor.is-focused)
     -1:03.4.0 editor:newline (atom-text-editor.editor.is-focused)
     -1:01.7.0 docblockr:parse-enter (atom-text-editor.editor.is-focused)
     -0:50.7.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:50.3.0 docblockr:parse-enter (atom-text-editor.editor.is-focused)
     -0:50.3.0 editor:newline (atom-text-editor.editor.is-focused)
     -0:50 core:move-up (atom-text-editor.editor.is-focused)
  2x -0:47.1.0 core:backspace (atom-text-editor.editor.is-focused.autocomplete-active)
     -0:44 core:move-up (atom-text-editor.editor.is-focused)
     -0:43.5.0 docblockr:parse-enter (atom-text-editor.editor.is-focused)
     -0:43.5.0 editor:newline (atom-text-editor.editor.is-focused)
  2x -0:40.4.0 docblockr:parse-enter (atom-text-editor.editor.is-focused)
     -0:10 core:backspace (atom-text-editor.editor.is-focused)
     -0:00.5.0 docblockr:parse-enter (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "background-tips",
      "feedback",
      "welcome",
      "atom-alignment"
    ],
    "themes": [
      "proton",
      "monokai"
    ],
    "autoHideMenuBar": true
  },
  "docblockr": {
    "per_section_indent": true
  }
}
```

### Installed Packages

```coffee
# User
Stylus, v1.0.0
atom-jsfmt, v0.6.0
autocomplete-paths, v1.0.2
autocomplete-plus, v2.14.2
autocomplete-snippets, v1.6.1
color-picker, v1.7.0
docblockr, v0.7.1
editorconfig, v1.0.0
file-icons, v1.5.5
highlight-selected, v0.9.2
language-clojure, v0.14.0
language-jade, v0.4.0
language-lua, v0.9.2
language-pegjs, v0.5.0
linter, v0.12.2
linter-coffeelint, v0.2.3
linter-jshint, v0.1.4
linter-jsonlint, v0.1.3
linter-php, v0.0.12
minimap, v4.8.0
minimap-highlight-selected, v4.2.3
monokai, v0.14.0
permute, v0.1.0
pretty-json, v0.3.3
preview-tabs, v0.1.6
project-manager, v1.15.5
proton, v3.0.0
regex-railroad-diagram, v0.7.4
tidy-markdown, v0.2.2
wakatime, v4.0.9

# Dev
No dev packages
```

**Comment by jayflo** [comment]
I have just run across this same issue.

OS: Windows 7
Atom: v 0.198.0
Docblockr: v 0.7.1

**Comment by malko** [comment]
should be fixed in next release by commit 243fe41
